routes/votings/list-of-votings-f.js

```diff
@@ -0,0 +1,29 @@
+const knex = require("../../knex");
+
+function createList(list){
+    return knex("list_of_votings").insert(list);
+};
+
+function getAllLists() {
+    return knex("list_of_votings").select("*");
+};
+
+function getLists(idOfUser) {
+    return knex("list_of_votings").where("id_of_user", idOfUser).select("*");
+};
+
+function deleteList(id) {
+    return knex("list_of_votings").where("primary_key", id).del();

```

Comment:
A similar problem as with the `foreign_key` column name above. The collumn name should reflect the meaning and not the type of the column. Usually primary key columns are named `id`, but if you for example wanted to identify users by email, then the `email` field of the `users` table can be a primary key

Reply:
This function is not final. should i change the table?

Reply:
changed